src/box/space.c

```diff
@@ -813,12 +813,12 @@ space_run_replace_triggers(struct event *event, struct txn *txn,
 			struct xrow_header *row = stmt->row;
 			assert(row != NULL && row->header != NULL);
 			func_adapter_push_msgpack(trigger, &ctx, row->header,
-						  row->header_end);
+						  row->header_end, NULL);
 			assert(row->bodycnt == 1);
 			const char *body = row->body[0].iov_base;
 			const char *body_end = body + row->body[0].iov_len;
 			func_adapter_push_msgpack(trigger, &ctx, body,
-						  body_end);
+						  body_end, NULL);
```

Comment:
We could add translation to the space recovery triggers (as a separate commit).

Reply:
Done.